modules/mongodb/src/test/java/org/testcontainers/containers/MongoDBContainerTest.java

```diff
@@ -36,7 +38,7 @@ public void shouldExecuteTransactions() {
 
     private void executeTx(MongoDBContainer mongoDBContainer) {
         final String mongoRsUrl = mongoDBContainer.getReplicaSetUrl();
-        assertThat(mongoRsUrl).isNotNull();
+        Assertions.assertThat(mongoRsUrl).isNotNull();
```

Comment:
no need to add Assertions here

Reply:
i think this was for my debugging actually but it didn't leap out at me when I was cleaning stuff up.  will remove.